.github/workflows/test.yaml

```diff
@@ -39,4 +39,5 @@ jobs:
         with:
           dry: True
           prettier_options: --write .
+          prettier_version: 2.8.4
```

Comment:
did this solve the issue you were having on friday?

Reply:
yes, this solved it. I just added a comment to it